sources/platform/actors/development/builds_and_runs/builds.md

```diff
@@ -1,34 +1,65 @@
+- Timeout of _1800_ seconds
+- Memory: `4096 MB`
 
-By default, the build has a timeout of 300 seconds and consumes 4096 MB (2048 MB on the free plan) of memory from the user's memory limit. See the [Resource limits](../../running/index.md) section for more details.
+Check out [Resource limits](../../running/index.md) section for more details.
 
 ## Versioning
 
-In order to enable active development, the Actor can have multiple versions of the source code and associated settings, such as the **Base image** and **Environment**. Each version is denoted by a version number of the form `MAJOR.MINOR`; the version numbers should adhere to the [Semantic Versioning](https://semver.org/) logic.
+To support active development, Actors can have multiple versions versions of source code and associated settings, such as the base image and environment. Each version is denoted by a version number of the form _MAJOR.MINOR_; following [Semantic Versioning](https://semver.org/) principles.
 
-For example, the Actor can have a production version **1.1**, a beta version **1.2** that contains new features but is still backward compatible, and a development version **2.0** that contains breaking changes.
+For example, an Actor might have:
+
+- Production version _1.1_
+- Beta version _1.2_ that contains new features but is still backward compatible
+- Development version _2.0_ that contains breaking changes.
 
 ## Tags
 
-When running the Actor, the caller needs to specify which build should actually be used. To simplify this process, the builds can be associated with a tag such **latest** or **beta**, which can be used instead of the version number when running the Actor. The tags are unique - only one build can be associated with a specific tag.
+Tags simplify the process of specifying which build to use when running an Actor Instead of using a version number, you can use a tag such as _latest_ or _beta_. Tags are unique, meaning only one build can be associated with a specific tag.
+
+To set a tag for builds of a specific Actor version:
 
-To set a tag for builds of a specific Actor version, set the **Build tag** property. Whenever a new build of the version is successfully finished, it is automatically assigned the tag. By default, the builds are set to the **latest** tag.
+1. Set the `Build tag` property.
+2. When a new build of that version is successfully finished, it's automatically assigned the tag.
+
+By default, the builds are set to the _latest_ tag.
 
 ## Cache
 
-By default, the build process pulls the latest copies of all necessary Docker images and builds each new layer of Docker images from scratch. To speed up the builds triggered via API, you can add **useCache=1** parameter. See the API reference for more details.
+To speed up builds triggered via API, you can use the `useCache=1` parameter. This instructs the build process to use cached Docker images and layers instead of pulling the latest copies and building each layer from scratch.
```

Comment:
```suggestion
To speed up builds triggered via API, you can use the `useCache=1` parameter. This instructs the build process to use cached Docker images and layers instead of pulling the latest copies and building each layer from scratch. Note that the cached images and layers might not always be available on the server building the image, the `useCache` parameter only functions on a best-effort basis.
```